src/app/app.component.ts

```diff
@@ -424,15 +438,16 @@ export class AppComponent implements AfterViewInit {
   }
 
   async onOpenNode() {
-    // const nodeToOpen = this.entrySelected;
     await this.lfRepositoryBrowser?.nativeElement?.openSelectedNodesAsync();
   }
 
   async onClickBrowse() {
     this.expandFolderBrowser = true;
-    this.lfRepoTreeNodeService.columnIds = this.allPossibleColumns.map(
-      (columnDef) => columnDef.id
-    );
+    if (this.lfRepoTreeNodeService) {
+      this.lfRepoTreeNodeService.columnIds = this.allPossibleColumns.map(
```

Comment:
see if we can use ?

Reply:
I don't think so b/c we are setting the value instead of getting it.